src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -498,6 +498,14 @@ private function registerHttpCache(ContainerBuilder $container, array $config, X
 
         $loader->load('http_cache_tags.xml');
 
+        // Load purger
+        $purger = $config['http_cache']['invalidation']['purger'];
+        $container->setAlias('api_platform.http_cache.purger', 'api_platform.http_cache.purger.'.$purger);
```

Comment:
Can you use `"api_platform.http_cache.purger.$purger"` to trigger https://blog.blackfire.io/php-7-performance-improvements-encapsed-strings-optimization.html

Reply:
Oh nice one! I've read a lot about opcode optim but must have missed that one! Done in f4aec2ec81b4e56111465cbd2d1a7196249dfd4a.